### DISCUSSION TITLE
Server

### DISCUSSION THREAD

**Discussion by rjkproductions** [apohl79/audiogridder#601] [Q&A]
I tried to use a Macbook Air on a wireless network as a server, but it would work with Pro Tools. So, I’m looking at refurbished Mac Minis which will be connected to my iMac for production and mixing purposes. I’ve found a number of various models for sale, and would like to know what the minimum and optimal capacity needed before purchasing.

**Comment by jaacMusic** [comment]
Using a wireless network could bring a lot of latency issues and drops. The type of machine model/hardware all depends on the type of plugins you planning to run. There are plugins out there that with 5 instances can take a powerful system to a crawl.

**Reply by rjkproductions** [reply]
Mostly Acustica plugins as they are pretty CPU intensive. I've seen some affordable used Mac Minis but want to make sure they would be adequate. I want to spend as little as possible, but also don't want to buy something that won't work and waste my money. Thanks
